mixer/cmd/build.go

```diff
@@ -279,7 +270,10 @@ func init() {
 	RootCmd.AddCommand(buildCmd)
 
 	buildBundlesCmd.Flags().BoolVar(&buildFlags.noSigning, "no-signing", false, "Do not generate a certificate to sign the Manifest.MoM")
-	buildBundlesCmd.Flags().BoolVar(&builder.UseNewChrootBuilder, "new-chroots", false, "EXPERIMENTAL: Use new implementation of build chroots")
+	unusedBoolFlag := false
+	buildBundlesCmd.Flags().BoolVar(&unusedBoolFlag, "new-chroots", false, "")
+	_ = buildBundlesCmd.Flags().MarkHidden("new-chroots")
+	_ = buildBundlesCmd.Flags().MarkDeprecated("new-chroots", "this flag is no longer used")
```

Comment:
I would prefer if these were descriptive of why. It's already going to report the flag as "deprecated", so just saying it's no longer used feels redundant. Perhaps `"New chroot functionality is now the standard behavior. This flag is obsolete and will be removed."`, or something similar.